Source/Csla.Blazor.Test/Fakes/FakePerson.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+using Csla.Blazor.Test.Rules;
+
+namespace Csla.Blazor.Test.Fakes
+{
+  [Serializable()]
+  public class FakePerson : BusinessBase<FakePerson>
+  {
+    public static PropertyInfo<string> FirstNameProperty = RegisterProperty<string>(c => c.FirstName);
```

Comment:
This is a small thing, but the new CSLA v5 snippets and recommendation is to exchange the `c=>c.Name` lambda for `nameof(Name)`

Reply:
Ah, of course. I saw your issue about the potential change, but obviously failed to take it on board!

Ironically, I couldn't remember the exact syntax - I code generate at work, so I never write it by hand - so I copied this from somewhere else in the codebase.

I agree that the code should stay to stay up to date and be an example to others, so I'll create an issue for this and fix both this instance and the example of the old syntax I copied.